scripts/create-release-pr.sh

```
@@ -15,99 +65,63 @@ command -v jq >/dev/null 2>&1 || {
   echo "gh auth login を実行してGitHubに認証してください" >&2
   exit 1
 fi
 
-REPO=$(gh repo view --json nameWithOwner -q .nameWithOwner)
+git fetch origin "$BASE_BRANCH" "$HEAD_BRANCH" >/dev/null
 
-existing_pr=$(gh pr list --state open --base "$BASE_BRANCH" --head "$HEAD_BRANCH" --json number --jq '.[].number' 2>/dev/null || true)
-if [[ -n "${existing_pr:-}" ]]; then
-  echo "既に #$existing_pr のリリースPRが開かれています。" >&2
-  exit 0
+last_main_merge=$(git log origin/${BASE_BRANCH} --merges -1 --pretty=format:'%cd' --date=iso8601)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)